torch/_inductor/fx_passes/ddp_fusion.py

```diff
@@ -80,6 +84,17 @@ class CommBlock:
     outputs: Set[fx.Node]
 
 
+def is_with_effects_wait_tensor(node: torch.fx.Node) -> bool:
+    return is_with_effects_op(node, torch.ops._c10d_functional.wait_tensor.default)
+
+
+def is_wait_tensor(node: torch.fx.Node) -> bool:
+    return (
+        node.op == "call_function"
+        and node.target == torch.ops._c10d_functional.wait_tensor.default
```

Comment:
nit: you can remove `node.op == "call_function"` and just test `node.target == torch.ops._c10d_functional.wait_tensor.default`